framework/route_wrappers.py

```diff
@@ -0,0 +1,61 @@
+# Stdlib
+from functools import wraps
+import json as _json
+
+# External Libraries
+from flask import Response, abort, request
+
+# Sayonika Internals
+from framework.objects import auth_service
+
+__all__ = ("json", "requires_keycloak_login", "requires_keycloak_admin")
+
+
+def json(func):
+    """
+    Wraps a function to return a unified JSON format
+    """
+
+    @wraps(func)
+    def inner(*args, **kwargs):
+        response = func(*args, **kwargs)
+        text = response.response[0].decode()
+        try:
+            data = _json.loads(text)
+        except _json.JSONDecodeError:
+            data = text
+
+        result = _json.dumps({
+            "result": data,
+            "status": response._status_code,  # flake8: noqa pylint: disable=protected-access
```

Comment:
Do we really need to explicitly say the status in the body? People can just check the response status.

Reply:
We do have a use for it for the Server Status colored circles.

Reply:
Most "professional" APIs have similar responses so it seemed only natural to me to copy the response format. This way we can guarantee: 

- you can access the data through "result" when success == True
- you can access the error through "error" when success == False
- for more complex / less usable libraries, the status code is easily accessible
- clients can more easily tell if the request passed or not with the "success" field

Reply:
I think @martmists has a point here, we don't need to rely too much on HTTP/S codes just for it.